### ISSUE TITLE
Set timeout

### ISSUE THREAD

**Issue by skrosoft** [Rentlio/SwiftRestModel#16]
Hello,

Is there a way to set the max timeout?

Thank you.

**Comment by jurajhilje** [comment]
No, not at this moment, but i could implement it as a option parameter in a future release.
